docs/builds/guides/frameworks/angular.md

```diff
@@ -262,6 +262,46 @@ export class MyComponent {
 </button>
 ```
 
+### `watchdog`
+
+An instance of the {@link module:watchdog/contextwatchdog~ContextWatchdog `ContextWatchdog`} class that is responsible for providing the same context to the editor instances and restarting the whole structure in case of crashes.
```

Comment:
```suggestion
An instance of the {@link module:watchdog/contextwatchdog~ContextWatchdog `ContextWatchdog`} class that is responsible for providing the same context to multiple editor instances and restarting the whole structure in case of crashes.
```